src/stories/components/Branding/Branding.js

```diff
@@ -0,0 +1,52 @@
+import './styles.scss';
+
+import React from 'react';
+import PropTypes from 'prop-types';
+// TODO: replace the import below with the actual logo once it's chosen
+import Logo from '../../../images/Polygon.png';
```

Comment:
@fernandowinfield let's use an SVG file instead of PNG, that way it won't lose quality. You can export it as an SVG directly from Sketch.